src/test-reader/sets-builder/index.ts

```diff
@@ -1,7 +1,6 @@
 import path from "path";
-import * as globExtra from "glob-extra";
 import _ from "lodash";
-
+import * as globExtra from "../../bundle/glob-extra";
```

Comment:
What part of the PR is responsible for creating "../../bundle/glob-extra"?
I only see "bundle/@wdio-utils" and "bundle/cjs"